vl-template.js

```diff
@@ -1,16 +1,12 @@
 import{VlElement,define}from"/node_modules/vl-ui-core/vl-core.js";export class VlTemplate extends VlElement(HTMLElement){constructor(){super(`
-            <style>
-                @import '/node_modules/vl-ui-template/style.css';
-            </style>
-            <vl-header>
-              <slot name="header"></slot>
-            </vl-header>
-            <div class="vl-page">
-              <main class="vl-main-content">
-                <slot name="main"></slot>
-              </main>
-            </div>
-            <vl-footer>
-              <slot name="footer"></slot>
-            </vl-footer>
-        `)}};define("vl-template",VlTemplate);
\ No newline at end of file
+        <style>
+            @import '/node_modules/vl-ui-template/style.css';
+        </style>
+        <vl-header>header</vl-header>
+        <div class="vl-page">
+          <main class="vl-main-content">
+            <slot></slot>
+          </main>
+        </div>
+        <vl-footer>footer</vl-footer>
```

Comment:
ok, dus deze component steekt wel degelijk de header en footer er in. dan snap ik in de test hierboven niet waarom ge ook nog ne keer zelf die header er moet in steken ...

Reply:
De template.js is gebuilde code dus wordt pas bij release gebuild. Ik heb deze tijdelijk manueel gebuild om te kunnen gebruiken. Deze code kan dus steeds genegeerd worden.